scripts/local-dev/setup-rbe.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+# A script for automatically configuring a user's local dev
+# environment to use Remote Build Execution.
+
+# Determine the root directory of the Angular github repo.
+GITHUB_DIRECTORY_ROOT=$(git rev-parse --show-toplevel 2> /dev/null);
+if [[ $? -ne 0 ]]; then
+  echo "This command must be run from within the cloned \"angular/components\" repository."
+  exit 1;
+fi
+
+# Confirm gcloud installed and available as a command.
+if [ ! -x "$(command -v gcloud)" ]; then
+  echo "gcloud command is not available.  Please install gcloud before continuing."
```

Comment:
Could this script install `gcloud`?

Reply:
We can't install `gcloud` because dependent on your platform we have to install it differently.